router/transformer/transformer.go

```diff
@@ -279,6 +248,50 @@ func (trans *HandleT) Setup() {
 	trans.transformerResponseTransformRequestTime = stats.NewStat("router.transformer_response_transform_time", stats.TimerType)
 }
 
+func (trans *HandleT) makeHTTPRequest(ctx context.Context, url string, payload *strings.Reader) ([]byte, int, error) {
+	var respData []byte
+	var respCode int
+	payload.Seek(0, io.SeekStart)
```

Comment:
@utsabc You can remove `.Seek` here & use as @lvrach suggested. This is more from a readability point of view. Kindly make these changes

Reply:
Yes addressed this